README.md

```diff
@@ -1,2 +1,31 @@
-# wg
-Coordination repository of the Game Development Working Group 
+# Rust game development working group
+
+The game development working group's main purpose is to make Rust the default language option for game developers.
```

Comment:
This goal sounds too ambitious :thinking:

Reply:
I tend to agree, it's a little bit too far-reaching. How about:

> (...) to make Rust a first-class tool for game developers.

Reply:
"first-class" is a bit vague. Perhaps "competitive with C++"?

Reply:
As the one-liner description of our group’s purpose it’s fine to speak in the abstract here.

Reply:
I really think we could pull it of even with the default. :D, but I can put first-class there.